.github/workflows/pull-request.yml

```diff
@@ -21,9 +19,7 @@ jobs:
           ref: ${{ github.event.pull_request.head.ref }}
 
       - name: Fetch target
-        env:
-          BASE_REF: ${{ github.event.pull_request.base.ref }}
-        run: git fetch origin ${{ env.BASE_REF }}
+        run: git fetch origin ${{ github.event.pull_request.base.ref }}
```

Comment:
@TheOrangePuff I think this step was copied from an old workflow. I don't think it's actually needed but keep to hear your opinion.

Reply:
This is required for nx `affected` calculations to work, isn't it?

Reply:
That ring a bell. I checked again and did some test. This is redundant code. We only need the following part for `affected` commands to work:
```
- name: Derive appropriate SHAs for base and head for `nx affected` commands
        uses: nrwl/nx-set-shas@dbe0650947e5f2c81f59190a38512cf49126fe6b # v4
```